tests/integration/test_dynamodb.py

```diff
@@ -1016,16 +999,12 @@ def test_batch_write_items(self):
         )
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-        # clean up
-        dynamodb.delete_table(TableName=table_name)
-
-    def test_dynamodb_stream_records_with_update_item(self, dynamodb):
+    def test_dynamodb_stream_records_with_update_item(self, dynamodb_client, dynamodb_create_table):
```

Comment:
```suggestion
    def test_dynamodb_stream_records_with_update_item(self, dynamodb_client, dynamodb_create_table, dynamodb):
```
You would still need there the `dynamodb` fixture otherwise you get a failure at line 1011 (the test still calls that fixture).